src/main/java/spoon/reflect/declaration/CtTypeInformation.java

```diff
@@ -82,11 +82,21 @@
 	boolean isLocalType();
 
 	/**
+	 * Returns true if this type is an class. Returns false for others including `enum`.
```

Comment:
Returns true if this type is a class. Returns false for others (enum, interface, generics, annotation).